### DISCUSSION TITLE
Groupe Utilisateur & Synapps Affichage / Cacher un acteur

### DISCUSSION THREAD

**Discussion by FlorentAdell** [witsa/synapps#642] [Q&A]
Bonjour est-ce que c'est possible d'afficher ou cacher des valeurs ou des icones en fonction du groupe de l'utilisateur ? 

Exemple : Groupe 1 = Visu courbes 
                Groupe 2 = Visu courbes + modifications des consignes (qui étaient cacher en groupe 1)

**Comment by Raphwit** [comment]
Bonjour, il est possible d'accéder au niveau de l'utilisateur via les liaisons.

![image](https://github.com/witsa/synapps/assets/93767525/a1d703f8-8cdc-4e00-8002-82b1769d549b)

En revanche pour ce qui est des groupes je ne suis pas sûr qu'ils soient accessibles peut être @manufitoussiwit connait un moyen ..

**Comment by manufitoussiwit** [comment]
Oui, le moyen le plus simple est de se fier au champ `Ecriture ?` et `Lecture?` d'une ressource.

![image](https://github.com/witsa/synapps/assets/35595723/e7c14a91-d2ea-4e40-b0e1-3c415ce26b86)

En effet, ces champs seront vrai ou faux en fonction des droits de l'utilisateur. Cela suit son niveau de profil et son appartenance aux groupes.
Donc pas besoin de se prendre trop la tête avec les groupes :-) : *Il suffit de réaliser une simple liaison*.

Par exemple, on peut envisager de lier le champ `Lecture ?` d'une ressource qui appartient au groupe 2 à la visibilité d'un bouton de menu. Ainsi, si l'utilisateur a accès en lecture au groupe 2, il verra le menu. Dans le cas contraire, il ne le verra pas.

**Reply by FlorentAdell** [reply]
Merci Manu, je vais essayer ça dans la journée !
